UndertaleModTool/Scripts/Community Scripts/FixAlphysLabCrashAndroid.csx

```diff
@@ -0,0 +1,18 @@
+// Made by GitMuslim
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Linq;
+using UndertaleModLib.Util;
+
+EnsureDataLoaded();
+
+if (Data?.GeneralInfo?.DisplayName?.Content.ToLower() != "undertale")
```

Comment:
```suggestion
if (Data.GeneralInfo?.DisplayName?.Content.ToLower() != "undertale")
```
Because `EnsureDataLoaded()` already checks if `Data` is `null`.